android/maplibreui/src/main/java/com/stadiamaps/ferrostar/maplibreui/runtime/NavigationCamera.kt

```diff
@@ -6,22 +6,34 @@ import androidx.compose.ui.platform.LocalConfiguration
 import com.maplibre.compose.camera.MapViewCamera
 import com.maplibre.compose.camera.cameraPaddingFractionOfScreen
 
+sealed class NavigationActivity(val zoom: Double, val pitch: Double) {
+  /** The recommended camera configuration for automotive navigation. */
+  data object Automotive : NavigationActivity(zoom = 16.0, pitch = 45.0)
+
+  /** The recommended camera configuration for bicycle navigation. */
+  data object Bicycle : NavigationActivity(zoom = 18.0, pitch = 45.0)
+
+  /** The recommended camera configuration for pedestrian navigation. */
+  data object Pedestrian : NavigationActivity(zoom = 20.0, pitch = 10.0)
+}
+
 /**
  * The camera configuration for navigation. This configuration sets the camera to track the user,
- * with a bearing of 18 degrees and a pitch of 45 degrees. It automatically adjusts the padding
- * based on the screen size and orientation.
+ * with a zoom of 16.0 and a pitch of 45 degrees. It automatically adjusts the padding based on the
```

Comment:
Maybe something like this so we don't have to update docstrings all the time ;0 I think the exact numbers probably _shouldn't_ matter to users of this composable.

```suggestion
 * with a high zoom level and moderate pitch for a 2.5D isometric view. It automatically adjusts the padding based on the
```